src/client/app/translations/data.ts

```diff
@@ -339,8 +339,8 @@ const LocaleTranslationData = {
 		"meter.enabled": "Updates:",
 		"meter.endOnlyTime": "Only End Times:",
 		"meter.endTimeStamp": "End Time Stamp:",
-		"meter.minVal": "Minimum Reading Value Check",
-		"meter.maxVal": "Maximum Reading Value Check",
+		"minVal": "Minimum Reading Value Check",
```

Comment:
This should be placed alphabetically in all languages. Also, it does not follow the key naming conventions. I know the old value was something similar to this but lets use ``min.value``. Similar for max.